Swift/fibonacci/fibonacci.swift

```diff
@@ -1,12 +1,29 @@
-func fibonacci(n: Int) -> Int {
-var a = 0
-var b = 1
-for _ in 0..<n {
-let temp = a
-a = b
-b = temp + b
+func FibonacciSequence(n: Int) -> Int {
+	if n <= 1 {
+		return n
+	}
+
+	return FibonacciSequence(n: n-1) + FibonacciSequence(n: n-2)
+}
+
+let default_n = 10
+
+var n = default_n
+
+let arguments = CommandLine.arguments
+
+if (arguments.count > 1) {
+    n = Int(arguments[1]) ?? default_n
```

Comment:
This still does not do what either Zig, Grain or Rust (https://github.com/enarx/codex/pull/29) examples do.
For each argument passed it should print the fibbonaci number corresponding to that index.
All examples have to behave _exactly the same_, because we will use this for testing in CI and we will not write a custom test script for each language.